paddle/fluid/memory/allocation/cuda_allocator.cc

```diff
@@ -36,7 +37,7 @@ Allocation* CUDAAllocator::AllocateImpl(size_t size) {
   void* ptr;
   auto status = cudaMalloc(&ptr, size);
   if (UNLIKELY(status != cudaSuccess)) {
-    throw BadAlloc(string::Sprintf(
+    PADDLE_THROW_BAD_ALLOC(string::Sprintf(
```

Comment:
Why do you use `string::Sprintf` here?

Reply:
`string::Sprintf` can be removed here. Done.